sdv/single_table/ctgan.py

```diff
@@ -108,6 +108,33 @@ def __init__(self, metadata, enforce_min_max_values=True, enforce_rounding=True,
             'cuda': cuda
         }
 
+    def _estimate_num_columns(self, data):
+        """Estimate the number of columns that the data will generate.
+
```

Comment:
Can we add a line in the docstring explaining that continuous columns generate 11 columns and categorical create n where n is the number of unique categories (from the issue)? I think it'd be helpful to keep track of where all the magic numbers are coming from.